boards/riscv/tlsr9528a/tlsr9528a-common.dtsi

```diff
@@ -166,11 +166,11 @@
 };
 
 &gpioe {
+	interrupts = <38 1>;
```

Comment:
The PF IRQ was set in Zephyr, so the idea behind was to set the PE IRQ also in zephyr, because button functionality is the basic functionality, and if buttons defined in basic DTS, the IRQ also should be defined in basic DTS.